Gems/Atom/Feature/Common/Code/Source/Shadows/ProjectedShadowFeatureProcessor.cpp

```diff
@@ -265,23 +273,19 @@ namespace AZ::Render
         view->SetCameraTransform(Matrix3x4::CreateFromTransform(desc.m_transform));
 
         ShadowData& shadowData = m_shadowData.GetElement<ShadowDataIndex>(shadowProperty.m_shadowId.GetIndex());
-        shadowData.m_bias = (nearDist / farDist) * 0.1f;
+
+        // Adjust the manually set bias to a more appropriate range for the shader.
+        shadowData.m_bias = nearDist * shadowProperty.m_bias * 0.01f;
```

Comment:
Can you add comment on why use this equation to convert the shadowProperty.m_bias to shadowData.m_bias ?

Reply:
Expanded on the comment already there.